api/api.go

```diff
@@ -24,7 +24,7 @@ type RemoteCalls interface {
 
 	GetBillVendors(ctx context.Context, category string) (*[]model.Vendor, error)
 	GetBillVendorByID(ctx context.Context, vendorID string) (*model.Vendor, error)
-	GetBillProducts(ctx context.Context, vendorID string, category string) (*[]model.Product, error)
+	GetBillProducts(ctx context.Context, vendorID, category *string) (*[]model.Product, error)
```

Comment:
@odetolataiwo why you comot the type for the `vendorID`
Also both inputs are optional

Reply:
oh i understand